### PR TITLE
VACMS-17860: Telephone Migrations Integration

### PR DESCRIPTION
**Pull request by dsasser**
## Description
Integration branch for telephone migrations.

Relates to #17860

## Testing done
See each of the individual phone migration PRs for testing steps performed.

## QA steps
QA will be performed under https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19116

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.
- [ ] If there are field changes, front end output has been thoroughly checked.

### Select Team for PR review

- [x] `CMS Team`
- [x] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -50,6 +50,7 @@
 /docroot/modules/custom/va_gov_api                                  @department-of-veterans-affairs/accelerated-publishing
 /docroot/modules/custom/va_gov_backend                              # @department-of-veterans-affairs/platform-cms-drupal-engineers
 /docroot/modules/custom/va_gov_banner                               @department-of-veterans-affairs/public-websites-cms
+/docroot/modules/custom/va_gov_batch                                @department-of-veterans-affairs/public-websites-cms
```
Comment (jv-agile6) [inline .github/CODEOWNERS:53]: batch modules and libphonenumber dependencies added

File: config/sync/core.entity_form_display.node.health_care_local_facility.default.yml [hunk 2]
```diff
@@ -25,6 +25,7 @@ dependencies:
     - field.field.node.health_care_local_facility.field_region_page
     - field.field.node.health_care_local_facility.field_supplemental_status
     - field.field.node.health_care_local_facility.field_supplemental_status_more_i
+    - field.field.node.health_care_local_facility.field_telephone
```
Comment (jv-agile6) [inline config/sync/core.entity_form_display.node.health_care_local_facility.default.yml:28]: health_care_local_facility.field_telephone added

File: config/sync/core.entity_form_display.node.health_care_local_facility.default.yml [hunk 3]
```diff
@@ -369,6 +373,32 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  field_telephone:
```
Comment (jv-agile6) [inline config/sync/core.entity_form_display.node.health_care_local_facility.default.yml:376]: field_telephone configs added

File: config/sync/feature_toggle.features.yml [hunk 4]
```diff
@@ -59,3 +59,4 @@ features:
   feature_next_build_content_vha_facility_nonclinical_service: FEATURE_NEXT_BUILD_CONTENT_VHA_FACILITY_NONCLINICAL_SERVICE
   feature_next_build_content_vet_center_mobile_vet_center: FEATURE_NEXT_BUILD_CONTENT_VET_CENTER_MOBILE_VET_CENTER
   feature_next_build_content_vet_center_outstation: FEATURE_NEXT_BUILD_CONTENT_VET_CENTER_OUTSTATION
+  feature_telephone_migration_v1: FEATURE_TELEPHONE_MIGRATION_V1
```
Comment (jv-agile6) [inline config/sync/feature_toggle.features.yml:62]: migration toggle added

File: docroot/modules/custom/va_gov_batch/src/cbo_scripts/MigratePhoneFieldToParagraph.php [hunk 5]
```diff
@@ -0,0 +1,179 @@
+<?php
```
Comment (jv-agile6) [inline docroot/modules/custom/va_gov_batch/src/cbo_scripts/MigratePhoneFieldToParagraph.php:1]: migration logic

File: docroot/modules/custom/va_gov_vamc/src/EventSubscriber/VAMCEntityEventSubscriber.php [hunk 6]
```diff
@@ -13,6 +13,7 @@
 use Drupal\core_event_dispatcher\Event\Entity\EntityUpdateEvent;
 use Drupal\core_event_dispatcher\Event\Entity\EntityViewAlterEvent;
 use Drupal\core_event_dispatcher\Event\Form\FormIdAlterEvent;
+use Drupal\feature_toggle\FeatureStatus;
```
Comment (jv-agile6) [inline docroot/modules/custom/va_gov_vamc/src/EventSubscriber/VAMCEntityEventSubscriber.php:16]: Feature toggle dependency added

File: tests/phpunit/va_gov_batch/unit/Class/MigratePhoneFieldToParagraphTest.php [hunk 7]
```diff
@@ -0,0 +1,64 @@
+<?php
```
Comment (jv-agile6) [inline tests/phpunit/va_gov_batch/unit/Class/MigratePhoneFieldToParagraphTest.php:1]: MigratePhoneFieldToParagraphTest unit test added

**Review by jv-agile6 — APPROVED** [review]
Reviewed the following code for form and logic. No defect were detected. Approval granted.

**Review by SnowboardTechie — DISMISSED** [review]
LGTM 👍🏻

**Review by edmund-dunn — DISMISSED** [review]
No issues here!

**Pull request merged**